source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PlaceholderSvg from './PlaceholderSvg';
+
+export default [{
+  name: "default",
+  component: (
+    <PlaceholderSvg width={600} height='300' />
```

Comment:
@elseloop you are correct about the new-tag template, that is an artifact of starting those files as JSON files, where double quotes are expected.